zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/utils/CamundaExporterITInvocationProvider.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.utils;
+
+import static io.camunda.exporter.config.ConnectionTypes.ELASTICSEARCH;
+import static io.camunda.exporter.config.ConnectionTypes.OPENSEARCH;
+import static java.util.Arrays.asList;
+
+import io.camunda.exporter.config.ConnectionTypes;
+import io.camunda.exporter.config.ExporterConfiguration;
+import io.camunda.search.connect.es.ElasticsearchConnector;
+import io.camunda.search.connect.os.OpensearchConnector;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.Extension;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolutionException;
+import org.junit.jupiter.api.extension.ParameterResolver;
+import org.junit.jupiter.api.extension.TestTemplateInvocationContext;
+import org.junit.jupiter.api.extension.TestTemplateInvocationContextProvider;
+import org.opensearch.testcontainers.OpensearchContainer;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+
+public class CamundaExporterITInvocationProvider
+    implements TestTemplateInvocationContextProvider, AfterAllCallback, BeforeAllCallback {
+
+  public static final String CONFIG_PREFIX = "camunda-record";
+  private final ElasticsearchContainer elsContainer =
+      TestSupport.createDefeaultElasticsearchContainer();
+  private final OpensearchContainer<?> osContainer = TestSupport.createDefaultOpensearchContainer();
+
+  private final List<AutoCloseable> closeables = new ArrayList<>();
+
+  private ExporterConfiguration getConfigWithConnectionDetails(
+      final ConnectionTypes connectionType) {
+    final var config = new ExporterConfiguration();
+    config.getIndex().setPrefix(CONFIG_PREFIX);
+    config.getBulk().setSize(1); // force flushing on the first record
+    switch (connectionType) {
+      case ELASTICSEARCH -> config.getConnect().setUrl(elsContainer.getHttpHostAddress());
+      case OPENSEARCH -> config.getConnect().setUrl(osContainer.getHttpHostAddress());
+      default -> throw new IllegalArgumentException("Unknown connection type: " + connectionType);
```

Comment:
🔧 connectionType is an enum, it can never hold an `unsupported` case. by adding a default, we are actually removing a compile time insurance that people won't add a new connection type without supporting configuring it.
So I suggest to remove the default here.